Description: Fix libbacktrace build
 rustc's copy (which we're using instead, see override_dh_auto_configure) does
 not have config.guess so the build fails. but the underlying problem is that
 the logic here is totally wrong.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
Comment: already fixed upstream https://github.com/alexcrichton/backtrace-rs/blob/master/backtrace-sys/build.rs
diff --git a/src/vendor/backtrace-sys/build.rs b/src/vendor/backtrace-sys/build.rs
index f29c704da..a533489f7 100644
--- a/src/vendor/backtrace-sys/build.rs
+++ b/src/vendor/backtrace-sys/build.rs
@@ -80,7 +80,8 @@ fn main() {
                 .arg("--disable-shared")
                 .arg("--disable-host-shared")
                 .arg(format!("--target={}", target))
-                .arg(format!("--host={}", host)),
+                .arg(format!("--host={}", target))
+                .arg(format!("--build={}", host)), // GNU vs LLVM terminology
         "sh");
     run(Command::new("make")
                 .current_dir(&dst)
